R/install-dev.bat

```diff
@@ -25,3 +25,8 @@ set SPARK_HOME=%~dp0..
 MKDIR %SPARK_HOME%\R\lib
 
 R.exe CMD INSTALL --library="%SPARK_HOME%\R\lib"  %SPARK_HOME%\R\pkg\
+
+REM Zip the SparkR package so that it can be distributed to worker nodes on YARN
+pushd %SPARK_HOME%\R\lib
+jar cfM "%SPARK_HOME%\R\lib\sparkr.zip" SparkR
```

Comment:
@sun-rui -- this should be `jar.exe` instead of `jar`. The other thing is that `jar.exe` is only available in the JDK and not in the JAR version. So sometimes this may not be in the `PATH`. There are a couple of options for things we can do here

1. We can use `%JAVA_HOME%\bin\jar.exe` -- This might be more safer as users need to set JAVA_HOME for the compilation to work correctly
2. Rtools [1] by default installs a zip utility [2] as `zip.exe`. At least on my machine running `zip.exe -r sparkr.zip SparkR` seems to work.

[1] http://cran.r-project.org/bin/windows/Rtools/
[2] http://www.info-zip.org/